tests/advancedAnalytics.spec.ts

```
@@ -0,0 +1,317 @@
+              await test.step(`Check ${analysis.calculation} object data in ${item.numerator}/${item.denominator} data in advancedAnalytics`, async () => {
+                expect(
+                  analysis.value,
+                  `Value in analytics (${analysis.value}) should be defined`
+                ).toBeDefined();
+
+                // TODO: udjust this tests after fixing https://kontur.fibery.io/Tasks/Task/insights-api-Nulls-in-value-and-quality,-median-is-on-20713
+
+                if (
+                  // analysis.calculation !== `stddev`
+                  // analysis.calculation !== `median`
+                  false
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)